website/server/controllers/api-v3/tasks.js

```diff
@@ -331,11 +335,12 @@ api.scoreTask = {
           user: user.profile.name,
           taskName: task.text,
         }),
+        groupId: group._id,
       });
 
       await Bluebird.all([groupLeader.save(), task.save()]);
 
-      return res.respond(200, {message: res.t('taskApprovalHasBeenRequested'), task});
```

Comment:
not sure this should be an error but I'd like to discuss it because maybe the 200 response isn't the best fit either